functions/src/image/save-valid-image/save-valid-image.test.ts

```diff
@@ -0,0 +1,155 @@
+import * as admin from 'firebase-admin'
+import * as test from 'firebase-functions-test'
+import * as functions from '../../index'
+import { Collections } from '../../model/collections'
+import { Image, ImageStatus, Mask } from '../../model/image'
+import { SaveValidImageRequest } from './model/save-image-request'
+
+describe('SaveValidImage', () => {
+  const db = admin.firestore()
+  const saveValidImageFunction = test().wrap(functions.saveValidImage)
+
+  beforeEach(async () => {
+    await test().firestore.clearFirestoreData({ projectId: 'aiscope-labelling-app-test' })
```

Comment:
Suggestion: the project id should be at some point a variable depending on which environment we are testing on.

Reply:
The tests are exclusively using the Firebase emulator, so this is the "local" environment emulating the "test".
I think it is not intended to be changeable.

Aligned here: https://github.com/theaiscope/GDD-backend/blob/9463aa5c76be880eeabe8c3f29bcdd571c803f1c/functions/package.json#L11